.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules/
 scratch/
 *.DS_Store
 Thumbs.db
+upload.sh
```

Comment:
Unsure why this has been added?
```suggestion
```
@TheGiddyLimit should we put in an .gitignore allowlist of filetypes for the img, pdf, font etc dirs?

Reply:
there's already a `scratch/` directory where one is free to dump one's random shell scripts

so, no